woocommerce/woocommerce-android
# [Login] Jetpack connection Part 1

Part of: #7299 

Requirements before merging:
- [x] Merging https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2522 and updating fluxc version commit hash.
- [x] Merging #7330 

### Description
This PR adds the first part of handling Jetpack connection during login with site credentials, it adds a button to the account mismatch error that allows starting the Jetpack connection in a WebView, and it detects the end of the connection.
The next steps of the connection will be handled in next PRs.

### Testing instructions

1. Prepare a site that has two accounts, where one of them is connected to Jetpack.
2. Open the app, and sign in using the site address.
3. Use site credentials for login, and enter credentials of the non-connected account.
4. Click on "Connect Jetpack to your account"
5. Continue the connection on the WebView.
6. Confirm the WebView is closed after finishing the connection, and the logcat contains a message: "Jetpack Connected"

### Images/gif
https://user-images.githubusercontent.com/1657201/188423223-fd1c84cd-2ce5-4cb0-9729-e2bb29d5d202.mov
